File: src/main/java/com/example/deposit/config/ObjectMapperConfig.java
package com.example.deposit.config;

import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

/**
 * ObjectMapperConfig.
 */
@Configuration
@ConfigurationProperties(prefix = "kafka")
public class ObjectMapperConfig {
    @Bean
    public ObjectMapper objectMapper() {
        return new ObjectMapper()
                .registerModule(new JavaTimeModule());
    }
}

File: ms-ppcd-example-app/src/main/kotlin/ru/vtb/ppcd/example/config/ObjectMapperConfig.kt
package org.example.ru.vtb.ppcd.example.config

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
open class ObjectMapperConfig {

    @Bean
    open fun objectMapper(): ObjectMapper =
        ObjectMapper()
            .registerModules(JavaTimeModule())
}